salt/utils/idem.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+"""
+Idem Support
+============
+
+This util provides access to an idem-ready hub
+
+.. versionadded:: Magnesium
+"""
+import logging
+import sys
+
+try:
+    import pop.hub
+
+    HAS_POP = True, None
+except ImportError as e:
+    HAS_POP = False, e
```

Comment:
Why not assign `False, str(e)`? All uses of `HAS_POP[1]` do a string conversion.

Just storing `e`, the stack frames and variables leading to this import are forced to persist for the lifetime of the program.